### PR TITLE
release 1.1.0

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

**Comment by olkornii** [comment]
+1, see: https://github.com/redhat-developer/intellij-kubernetes/issues/628

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -10,6 +10,18 @@
     </description>
     <change-notes>
         <![CDATA[
+        <h3>1.1.0</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/617">As a user I want to be able to open the web console to my kubernetes or openshift cluster</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:15]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/617">As a user I want to be able to open the web console for Kubernetes or OpenShift cluster</a></li>
```

File: README.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ It is available for install from the [JetBrains Marketplace](https://plugins.jet
  - Switching contexts and namespaces within Kubernetes
  - Follow logs from the connected cluster resource
  - Open a Terminal Shell to a Running Container in the connected cluster
+ - Open the Dashboard for a connected cluster
```
Comment (mohitsuman) [inline README.md:28]:
```suggestion
 - Open the Dashboard for the connected cluster (works on any Hybrid cloud cluster)
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
added few wording changes

**Comment by adietish** [comment]
plugin was submitted for approval: https://plugins.jetbrains.com/plugin/15921-kubernetes-by-red-hat/versions/stable/369015
image

**Pull request merged**